userguide/tutorials/email-notification-plugin.adoc

```diff
@@ -61,12 +63,36 @@ The email-notification-plugin requires some additional database tables. To creat
 == Plugin Configuration
 
-In order to use the email notification plugin, a tenant needs to be configured with the SMTP properties required for sending emails. Additionally, the tenant can also be configured with the events for which emails should be sent. In addition to the per-tenant configuration, the plugin also allows a more granular account-level configuration for the set of emails to be sent for the particular account. Thus, either the tenant or the account needs to be configured with the events for which the email needs to be sent.
+In order to use the email notification plugin, you need to configure either SMTP or AWS SES settings. Below are the steps to configure both:
+
+=== AWS SES Configuration
+To enable AWS SES for email notifications, add the following properties into killbill.properties.
+
+[source,bash]
+org.killbill.email.notification.via.ses=true
+org.killbill.mail.from=<Default sender's email>
+org.killbill.aws.region=<AWS region, default region is us-east-1>
+
+AWS SES requires authentication for sending emails. System-wide environment variables can be configured for all users by appending them to the /etc/environment file. Open the `/etc/environment` file in a text editor with root privileges, set the following variables:
+
+[source,bash]
+AWS_ACCESS_KEY_ID=<AWS access key ID>
+AWS_SECRET_ACCESS_KEY=<AWS access secret>
+
+
+Reboot the system to apply the changes.
+
+
+[NOTE]
+*Note:* Ensure that the credentials used have the necessary permissions, specifically `ses:SendEmail`, on the resource arn:aws:ses:<region>:<account>:identity/<sender's email>.
+
+=== SMTP Configuration
+A tenant needs to be configured with the SMTP properties required for sending emails. Additionally, the tenant can also be configured with the events for which emails should be sent. In addition to the per-tenant configuration, the plugin also allows a more granular account-level configuration for the set of emails to be sent for the particular account. Thus, either the tenant or the account needs to be configured with the events for which the email needs to be sent.
 
 The tenant/account configuration can be done by executing the required API endpoints via *cURL* as explained below:
 
 [[tenant-config]]
-. *Configure the tenant* using the following cURL command (Replace `<events>` with a comma-separated list of the events for which the emails need to be sent. For example, to configure the tenant to send emails for *INVOICE_CREATION* and *INVOICE_PAYMENT_SUCCESS*, specify `org.killbill.billing.plugin.email-notifications.defaultEvents=INVOICE_CREATION,INVOICE_PAYMENT_SUCCESS`):
+*Configure the tenant* using the following cURL command (Replace `<events>` with a comma-separated list of the events for which the emails need to be sent. For example, to configure the tenant to send emails for *INVOICE_CREATION* and *INVOICE_PAYMENT_SUCCESS*, specify `org.killbill.billing.plugin.email-notifications.defaultEvents=INVOICE_CREATION,INVOICE_PAYMENT_SUCCESS`):
```

Comment:
Is this on purpose (no more space) ?

Reply:
Is there a specific reason to get rid of the numbering?